### ISSUE TITLE
pd externals

### ISSUE THREAD

**Issue by giuliomoro** [BelaPlatform/Bela#621]
Over time I built several pd externals libraries for Bela, and I lost track of them. I will upload them here from now on, so that I don't have to build them again every time.

* pd-else v1.0-beta27 (uses pd-lib-builder)

How to:
`make PDINCLUDEDIR=/usr/local/include/libpd/`
it errors for `pic.c` as `IHEIGHT` cannot be found. Fixed with:
```
cc -DPD -I "/usr/local/include/libpd/" -DUNIX  -fPIC -Ishared -DHAVE_STRUCT_TIMESPEC -Wall -Wextra -Wshadow -Winline -Wstrict-aliasing -O3 -ffast-math -funroll-loops -fomit-frame-pointer -march=armv7-a -mfpu=vfpv3 -mfloat-abi=hard -o Classes/Source/pic.o -c Classes/Source/pic.c -DIHEIGHT=20
```
`make PDINCLUDEDIR=/usr/local/include/libpd/ PDLIBDIR=/root/Bela/projects/pd-externals/ install`
Built against:
```
ii  libpd-xenomai-3-dev                          0.11.0-Bela-1                     armhf        libpd for arm and xenomai-3. Has Pd 0.48
```
Download:
[else-v1.0-beta27.zip](https://github.com/BelaPlatform/Bela/files/4428489/else-v1.0-beta27.zip)

**Comment by csounder** [comment]
YES!  csound6~ works on the Bela!!!  Thanks for these instructions and for this build.  It would be great for you to share it and include it with the next release.  And maybe include a few csound~ examples in the examples "lightbulb" section.

NOTE:  Your installation instructions above are not correct in that they tell the installer to say "no" - and there is no "no".
There is "cancel".  In the fine print of the dialog box that appears when you try to install an external, one is advised to "upload" the .zip into a project and follow the steps from there - and this works perfectly.  You might want to update the installation instructions.  AND.... THANKS!!!!

**Comment by giuliomoro** [comment]
thanks for your feedback, I fixed the documentation above.

**Comment by csounder** [comment]
You might want to include/modify/embellish a few of these csound6~ examples by Victor Lazzarini to the Bela IDE
- it should be easy to add Trill sensors to some of them via PD too - and include these a models and examples.
- the models in the Bela IDE are such a great and inspiring starting point for all of us.

[csound6~_examples.zip](https://github.com/BelaPlatform/Bela/files/4873416/csound6._examples.zip)

https://www.dropbox.com/s/xjnjppfp16v9x9z/csound6~_examples.zip?dl=0

**Comment by tommyecho** [comment]
Hi Giulio! Thank you so much for building the manta external. I have a project that uses [comport] and [list-abs] (particularly the [list-l2s] abstraction). I'm wondering if you might be willing to build these for the bela? I can use them to wirelessly communicate between an augmented guitar device I built and my modular, so it would be incredibly helpful! Thanks for considering:)

**Comment by giuliomoro** [comment]
`list-abs` is just abstractions, so it should work out of the box. `comport` is not real-time safe, so I'd recommend using an OSC to serial bridge or something similar.
